<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlightController</class>
 <widget class="QMainWindow" name="FlightController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget{
	
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1,  stop:0 	rgb(1, 18, 71), stop:0.5 rgb(0, 58, 120), stop:1 rgb(1, 18, 71));
}

#centralwidget QLabel{
	color: white;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>473</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lbUsername">
         <property name="text">
          <string>Username</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLogout">
         <property name="text">
          <string>Logout</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="FlightManager">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabWelcome">
       <property name="styleSheet">
        <string notr="true">#tabWelcome{
	border: 3px solid white;
	background: rgb(2, 61, 116);
}</string>
       </property>
       <attribute name="title">
        <string>Welcome</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabList">
       <property name="styleSheet">
        <string notr="true">#tabList{
	border: 3px solid white;
	background: rgb(2, 61, 116);
}</string>
       </property>
       <attribute name="title">
        <string>Show List</string>
       </attribute>
       <widget class="QListWidget" name="lvPenerbangan">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>291</width>
          <height>321</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Penerbangan</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>10</y>
          <width>46</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Aviator</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>130</y>
          <width>46</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Crew</string>
        </property>
       </widget>
       <widget class="QListWidget" name="lvAviatorShow">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>30</y>
          <width>291</width>
          <height>91</height>
         </rect>
        </property>
       </widget>
       <widget class="QListWidget" name="lvCrewShow">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>150</y>
          <width>291</width>
          <height>91</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="lbDateShow">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>260</y>
          <width>46</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;date&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbPesawatShow">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>350</y>
          <width>291</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;AirPlane&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbDetailArrive">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>320</y>
          <width>291</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;Arrival&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbDetailDepart">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>280</y>
          <width>291</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;Departure&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>300</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>to</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnRefresh">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>360</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabScheduling">
       <property name="styleSheet">
        <string notr="true">#tabScheduling{
	border: 3px solid white;
	background: rgb(2, 61, 116);
}</string>
       </property>
       <attribute name="title">
        <string>Scheduling</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Date</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QDateEdit" name="deDate">
                <property name="displayFormat">
                 <string>yyyy/MM/dd</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Departure</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbBandaraBerangkat"/>
            </item>
            <item>
             <widget class="QTimeEdit" name="teJamBerangkat">
              <property name="displayFormat">
               <string>h:mm</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Arrival</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbBandaraDatang"/>
            </item>
            <item>
             <widget class="QTimeEdit" name="teJamDatang">
              <property name="displayFormat">
               <string>h:mm</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Airplane</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbPesawat"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Aviator</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QListWidget" name="lvAviatorList"/>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QPushButton" name="btnAviAdd">
                  <property name="text">
                   <string>Add</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnAviRemove">
                  <property name="text">
                   <string>Remove</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Crew</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QListWidget" name="lvCrewList"/>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QPushButton" name="btnCrewAdd">
                  <property name="text">
                   <string>Add</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnCrewRemove">
                  <property name="text">
                   <string>Remove</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnCreate">
            <property name="text">
             <string>Create</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnClear">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>649</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
